### PR TITLE
Fix build through addon-builder.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

File: package.sh [hunk 1]
```diff
@@ -20,6 +20,9 @@ rm -rf *.tgz package SHA256SUMS lib
 # Prep new package
 mkdir package lib
 
+# Make sure Cython is installed, as it's required to build numpy
+pip3 install Cython
```
Comment (rzr) [inline package.sh:24]: it wont harm , but for some reason I didnt need it, I am comparing my log to builder's one:

https://github.com/mozilla-iot/addon-builder/runs/657262548?check_suite_focus=true

https://github.com/mozilla-iot/addon-list/pull/825

Comment (mrstegeman) [reply package.sh:24]: Maybe you already had it installed?

**Review by rzr — COMMENTED** [review]
to be updated in v0.0.5

File: package.sh (hunk 1 above)
Comment (rzr) [reply package.sh:24]: no I checked there is no other cython on my py 3.7 , it's pulled by pip:

```
pi        1586  0.0  0.2   7676  2600 pts/2    S+   22:44   0:00                          \_ /bin/bash -e ./package.sh
pi        1595  2.2  3.4  40636 32512 pts/2    S+   22:44   0:10                              \_ /usr/bin/python3 /usr/bin/pip3 install -r requirements.txt -t lib --no-binary :all: --prefix
pi        1618  3.1  3.4  41388 33144 pts/2    S+   22:45   0:14                                  \_ /usr/bin/python3 -m pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-grru9kun --no-warn-script-location --no-binary :all: --only-binary :none: -i https://pypi.org/simple --extra-index-url https://www.piwheels.org/simple -- setuptools wheel Cython>=0.29.14
```


So I suspect a bug in pip: log says:
"You are using pip version 9.0.1, however version 20.1 is available."

raspbian:buster uses:
pip 18.1 from /usr/lib/python3/dist-packages/pip (python 3.7)

May I suggest to use "pip install --upgrade pip" instead ?

https://pip.pypa.io/en/stable/news/# 10.0.0b1 (2018-03-31) is minimal version to supports  

https://www.python.org/dev/peps/pep-0518/

https://github.com/numpy/numpy/blob/master/pyproject.toml


BTW, why building for 3.5 , addon supports only current version of gateway (py 3.7) ?

Comment (mrstegeman) [reply package.sh:24]: Ah, I see. I'm rebuilding the toolchains with an upgraded pip.

> BTW, why building for 3.5 , addon supports only current version of gateway (py 3.7) ?

The gateway doesn't enforce any particular python version requirements, so we try to support anything modern, e.g. 3.5+.

**Pull request closed without merging**